Content.Server/GameTicking/Rules/Components/ThiefRuleComponent.cs

```diff
@@ -0,0 +1,54 @@
+using Robust.Shared.Audio;
+using Robust.Shared.Prototypes;
+using Content.Shared.Roles;
+using Robust.Shared.Player;
+using Content.Shared.Preferences;
+
+namespace Content.Server.GameTicking.Rules.Components;
+
+/// <summary>
+/// Stores data for <see cref="ThiefRuleSystem/">.
+/// </summary>
+[RegisterComponent, Access(typeof(ThiefRuleSystem))]
+public sealed partial class ThiefRuleComponent : Component
+{
+    /// <summary>
+    /// A chance for this mode to be added to the game.
+    /// </summary>
+    [DataField]
+    public float RuleChance = 1f;
+
+    [DataField]
+    public ProtoId<AntagPrototype> ThiefPrototypeId = "Thief";
+
+    public Dictionary<ICommonSession, HumanoidCharacterProfile> StartCandidates = new();
+
+    [DataField]
+    public float MaxObjectiveDifficulty = 2.5f;
+
+    [DataField]
+    public int MaxStealObjectives = 10;
+
+    /// <summary>
+    /// Things that will be given to thieves
+    /// </summary>
+    [DataField]
+    public List<EntProtoId> StarterItems = new List<EntProtoId> { "ToolboxThief", "ClothingHandsChameleonThief" }; //TO DO - replace to chameleon thieving gloves whem merg
+
+    /// <summary>
+    /// All Thiefes created by this rule
+    /// </summary>
+    public readonly List<EntityUid> ThiefMinds = new();
+
```

Comment:
Should be a datafield, also thieves.

Reply:
![image](https://github.com/space-wizards/space-station-14/assets/96445749/81ef58a2-0d6e-4f6a-802e-38653d811930)
readonly datafield?